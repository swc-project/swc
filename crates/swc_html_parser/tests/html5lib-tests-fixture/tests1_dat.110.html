<table><colgroup><tbody><colgroup><tr><colgroup><td><colgroup></table><colgroup>