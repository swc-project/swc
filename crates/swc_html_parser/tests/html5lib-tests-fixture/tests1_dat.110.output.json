{
  "type": "Document",
  "span": {
    "start": 1,
    "end": 81
  },
  "mode": "quirks",
  "children": [
    {
      "type": "Element",
      "span": {
        "start": 0,
        "end": 0
      },
      "tagName": "html",
      "namespace": "http://www.w3.org/1999/xhtml",
      "attributes": [],
      "children": [
        {
          "type": "Element",
          "span": {
            "start": 0,
            "end": 0
          },
          "tagName": "head",
          "namespace": "http://www.w3.org/1999/xhtml",
          "attributes": [],
          "children": [],
          "content": null,
          "isSelfClosing": false
        },
        {
          "type": "Element",
          "span": {
            "start": 0,
            "end": 0
          },
          "tagName": "body",
          "namespace": "http://www.w3.org/1999/xhtml",
          "attributes": [],
          "children": [
            {
              "type": "Element",
              "span": {
                "start": 1,
                "end": 71
              },
              "tagName": "table",
              "namespace": "http://www.w3.org/1999/xhtml",
              "attributes": [],
              "children": [
                {
                  "type": "Element",
                  "span": {
                    "start": 8,
                    "end": 18
                  },
                  "tagName": "colgroup",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [],
                  "content": null,
                  "isSelfClosing": false
                },
                {
                  "type": "Element",
                  "span": {
                    "start": 18,
                    "end": 25
                  },
                  "tagName": "tbody",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [],
                  "content": null,
                  "isSelfClosing": false
                },
                {
                  "type": "Element",
                  "span": {
                    "start": 25,
                    "end": 35
                  },
                  "tagName": "colgroup",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [],
                  "content": null,
                  "isSelfClosing": false
                },
                {
                  "type": "Element",
                  "span": {
                    "start": 0,
                    "end": 0
                  },
                  "tagName": "tbody",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [
                    {
                      "type": "Element",
                      "span": {
                        "start": 35,
                        "end": 39
                      },
                      "tagName": "tr",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "attributes": [],
                      "children": [],
                      "content": null,
                      "isSelfClosing": false
                    }
                  ],
                  "content": null,
                  "isSelfClosing": false
                },
                {
                  "type": "Element",
                  "span": {
                    "start": 39,
                    "end": 49
                  },
                  "tagName": "colgroup",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [],
                  "content": null,
                  "isSelfClosing": false
                },
                {
                  "type": "Element",
                  "span": {
                    "start": 0,
                    "end": 0
                  },
                  "tagName": "tbody",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [
                    {
                      "type": "Element",
                      "span": {
                        "start": 0,
                        "end": 0
                      },
                      "tagName": "tr",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "attributes": [],
                      "children": [
                        {
                          "type": "Element",
                          "span": {
                            "start": 49,
                            "end": 53
                          },
                          "tagName": "td",
                          "namespace": "http://www.w3.org/1999/xhtml",
                          "attributes": [],
                          "children": [],
                          "content": null,
                          "isSelfClosing": false
                        }
                      ],
                      "content": null,
                      "isSelfClosing": false
                    }
                  ],
                  "content": null,
                  "isSelfClosing": false
                },
                {
                  "type": "Element",
                  "span": {
                    "start": 53,
                    "end": 63
                  },
                  "tagName": "colgroup",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [],
                  "content": null,
                  "isSelfClosing": false
                }
              ],
              "content": null,
              "isSelfClosing": false
            }
          ],
          "content": null,
          "isSelfClosing": false
        }
      ],
      "content": null,
      "isSelfClosing": false
    }
  ]
}
