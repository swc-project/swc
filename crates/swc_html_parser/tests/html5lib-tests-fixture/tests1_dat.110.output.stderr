(1,7): expected-doctype-but-got-start-tag
(1,52): unexpected-cell-in-table-body
(1,80): unexpected-start-tag-ignored